src/shared/varlink-systemd.c

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "alloc-util.h"
+#include "string-util.h"
+#include "varlink-systemd.h"
+#include "version.h"
+
+int varlink_set_info_systemd(sd_varlink_server *server) {
```

Comment:
I'd like to make this also internally call `sd_varlink_server_new()`, and rename this to e.g. `varlink_server_new()`, `varlink_server_new_systemd()`, `varlink_server_new_default()`, or so.